src/main/resources/static/front/css/flowA/A01-avatar.css

```
@@ -0,0 +1,757 @@
+.a01__log::-webkit-scrollbar-thumb {
+    background: var(--neutral-200);
+    border-radius: 2px;
+}
+
+.a01__msg {
+    max-width: 85%;
+    padding: 10px 14px;
+    border-radius: 16px;
+    font-size: 15px;
+    line-height: 1.45;
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)